.gitlab-ci.yml

```diff
@@ -50,6 +50,7 @@ lint:
 test:
   stage: test
   script:
+    - find packages/icon/src/react/icons -name "*.dist.tsx" -type f  -delete
```

Comment:
I think this should fix the problem if the first build passes and then I merge master on top. That's how #1313 failed.

Reply:
awesome find!